### PR TITLE
Update bug redeem point

### PR DESCRIPTION
**Pull request by zzybluebell**

### CODE REVIEW

**Review by HolmesJJ — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/AddTransactionCommand.java [hunk 1]
```diff
@@ -99,23 +99,23 @@ private static Member createEditedMember(Member memberToEdit, Transaction transa
...
-        Name updatedName = memberToEdit.getName();
-        Phone updatedPhone = memberToEdit.getPhone();
-        Email updatedEmail = memberToEdit.getEmail();
-        Address updatedAddress = memberToEdit.getAddress();
-        Timestamp updateTimestamp = memberToEdit.getTimestamp();
+        Name name = memberToEdit.getName();
+        Phone phone = memberToEdit.getPhone();
+        Email email = memberToEdit.getEmail();
+        Address address = memberToEdit.getAddress();
+        Timestamp timestamp = memberToEdit.getTimestamp();
```
Comment (Morrow1ndy) [inline src/main/java/seedu/address/logic/commands/AddTransactionCommand.java:106]: Good change.

File: src/main/java/seedu/address/logic/commands/RedeemCommand.java [hunk 2]
```diff
@@ -169,7 +169,7 @@ private static Member createToRedeemPointsMember(Member memberToRedeemPoints, Li
             throw new CommandException(MESSAGE_INVALID_POINTS_LESS_THAN_ZERO);
         }
         Point updatePoint = new Point(String.valueOf(oldPoint - toRedeemPoint));
-        return new Member(id, updatedName, updatedPhone, updatedEmail, updatedAddress, updateTimestamp, updateCredit,
-                updatePoint, updatedTransactions, updateReservations, updatedTags);
+        return new Member(id, name, phone, email, address, timestamp, credit,
+                updatePoint, transactions, reservations, tags);
     }
 }
```
Comment (Morrow1ndy) [inline src/main/java/seedu/address/logic/commands/RedeemCommand.java:175]: Overall LTGM.

**Review by Morrow1ndy — COMMENTED** [review]
Awesome

**Pull request merged**